src/borg/borg7.c

```diff
@@ -4280,7 +4280,7 @@ bool borg_play_magic(bool bored)
 				borg_keypress('t');
 			}
 
-			/* Hack -- Allow genocide spells */
+			/* Hack -- Allow spells that require selection of a monster type */
 			if (as->effect_index == EF_SELECT)
```

Comment:
If the code here is for the first cast of Banishment, shouldn't the test be for EF_BANISH rather than EF_SELECT?

Reply:
You are absolutely right.  Fixed it.

Reply:
Yes, if you look at files changed, this change should be in there.